docs/pages/ktable.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+
+  <DocsPageTemplate apiDocs>
+    <DocsPageSection title="Overview" anchor="#overview">
+      <p>
+        The <code>KTable</code> component is a flexible and customizable table component designed to handle a variety of data presentation needs within our application. It supports advanced features such as local sorting, sticky headers, keyboard navigation, and dynamic column resizing, making it suitable for both simple and complex data tables. 
+      </p>
+    </DocsPageSection>
+    <DocsPageSection title="Usage" anchor="#usage">
+      <!--Non-Sortable Table-->
+      <h3>Table without Sorting functionality</h3>
```

Comment:
```suggestion
      <h3>Table without sorting functionality</h3>
```

**[required] Would you decapitalize this title and all other titles on this page?** The convention on the KDS website is not to capitalize titles. It will help with consistency and orientation.

Total nitpick, shorter section titles, such as _"Without sorting"_, _"With built-in sorting"_, and (future) _"With externally sorted data"_ or similar, would do. On this page, it's clear we talk about the table.

You can see more details in the [Writing guidance](https://design-system.learningequality.org/writing/).